src/components/ebay-video/component.ts

```diff
@@ -58,6 +64,10 @@ interface VideoInput extends Omit<Marko.HTML.Video, `on${string}`> {
     clip?: any[];
     source: Marko.AttrTag<Marko.HTML.Source>;
     "report-text"?: AttrString;
+    "mute-text"?: AttrString;
+    "unmute-text"?: AttrString;
+    "fullscreen-text"?: AttrString;
+    "exit-fullscreen-text"?: AttrString;
     "spinner-timeout"?: number;
```

Comment:
Any reason these don't have an `a11y` prefix? I understand it would be a breaking change to also modify `report-text` to `a11y-report-text`, but maybe we can make an alias there?